src/pages/DelegationsPage/DelegationsAvailabilityTab/DelegationAvailabilityTab.tsx

```diff
@@ -7,19 +7,23 @@ import { AuthHooks } from '@/api/auth'
 import EditIcon from '@mui/icons-material/Edit'
 import { DelegationAvailabilityDrawer } from './DelegationAvailabilityDrawer'
 import { TenantHooks } from '@/api/tenant'
-import { hasTenantGivenProducerDelegationAvailability } from '@/utils/tenant.utils'
+import {
+  hasTenantGivenConsumerDelegationAvailability,
+  hasTenantGivenProducerDelegationAvailability,
+} from '@/utils/tenant.utils'
 
 export const DelegationsAvailabilityTab: React.FC = () => {
   const { t } = useTranslation('party', { keyPrefix: 'delegations.availabilityTab' })
   const { t: tCommon } = useTranslation('common')
   const { isAdmin } = AuthHooks.useJwt()
 
   const { data: activeTenant } = TenantHooks.useGetActiveUserParty()
-  const producerDelegationsAvailability = hasTenantGivenProducerDelegationAvailability(activeTenant)
 
+  const producerDelegationsAvailability = hasTenantGivenProducerDelegationAvailability(activeTenant)
   const isAvailableProducerDelegations = Boolean(producerDelegationsAvailability)
```

Comment:
I would change the function `hasTenantGivenProducerDelegationAvailability` to return a boolean instead of doing it outside